connectivity/FEATURE_BLE/source/generic/SecurityManagerImpl.cpp

```diff
@@ -537,18 +750,21 @@ ble_error_t SecurityManager::enableSigning(
                 cb->db_entry
             );
         } else {
+            tr_info("CSRK not available, pair to acquire it");
```

Comment:
```suggestion
            tr_info("CSRK not available, pairing to acquire it");
```
otherwise it implies user action was needed

Reply:
actually if you're slave you can't force that so maybe the trace should be in the if statement where the role is known